org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/util/IResourcesSetupUtil.java

```diff
@@ -237,9 +236,9 @@ protected void execute(IProgressMonitor monitor)
 					InterruptedException {
 				create(file.getParent());
 				file.delete(true, monitor());
-				try {
-					file.create(new StringInputStream(s, file.getCharset(true)), true, monitor());
-				} catch (UnsupportedEncodingException exc) {
+				try (InputStream stream = new StringInputStream(s, file.getCharset(true))) {
```

Comment:
That wouldn't be ncessary since it's a StringInputStream. But it's cleaner this way